lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -255,36 +257,15 @@ function update(id, body, callback) {
         if (error) {
             callback(error);
         } else {
-            const attributes = [
-                'url',
-                'apikey',
-                'type',
-                'service',
-                'subservice',
-                'description',
-                'trust',
-                'cbHost',
-                'timezone',
-                'timestamp',
-                'commands',
-                'lazy',
-                'attributes',
-                'staticAttributes',
-                'internalAttributes',
-                'explicitAttrs',
-                'expressionLanguage',
-                'defaultEntityNameConjunction',
-                'ngsiVersion'
-            ];
-
-            for (let i = 0; i < attributes.length; i++) {
-                if (body[attributes[i]] !== undefined) {
-                    group[attributes[i]] = body[attributes[i]];
+            attributeList.forEach((key) => {
```

Comment:
Until this `resource` was not able to be updated. With this resource could be updated. Not sure if is desirable.

Reply:
If you really want separately maintained lists for create and update, at least bring these out as constants at the head of the file. The current position where they are buried in the middle of the file means that they are more likely to get out of sync.

It also means a new array is created and destroyed for every update.  I'm happy to re-alter the functionality here, but I'd like a clear and definite plan.

Reply:
`apikey` and `resource` are keys to updae and delete a device group.
Sometime ago `apikey` and `resource` where not able to be modified. Currently `apikey` could be modified, so I don't see a reason to `resource` couldn't be modified.